src/stacks-hierarchy/Root_PurchaseConfirmationScreen/Root_PurchaseConfirmationScreen.tsx

```diff
@@ -169,6 +169,8 @@ export const Root_PurchaseConfirmationScreen: React.FC<Props> = ({
       )
     : t(PurchaseConfirmationTexts.travelDate.now);
 
+  const isCarnet = preassignedFareProduct.type === 'carnet';
```

Comment:
Not allowed to check on the type string. This is ment as a "foreign key" between the fare product type configs and preassigned fare products, and doesn't have any semantic meaning. If we wan't some logic especially for this product, it needs to be in the configs.

Reply:
how do we know a PreassignedFareProduct is carnet or not then? On a FareContract we can use TravelRight, not sure what to use on PreassignedFareProduct. 🤔

Reply:
The answer is that we shouldn't know if a PreassignedFareProduct is carnet or not. How we sell the project should be based on configuration in Firestore. So if a feature that is designed needs its own configuration we take a step back and evaluate if the feature is actually important enough that its worth the increase in configuration complexity, and have a talk with the designers and product owner about it.